Driver not compiling on  Kernel 6.12.4,  "aml_deinit" and "aml_init" undefined

Hi I did not succeed to compile the driver upon the kernel update on Arch. I get:
```
  MODPOST /home/werefkin/drivers/macbook12-bluetooth-driver/build/bluetooth/Module.symvers
ERROR: modpost: "aml_deinit" [/home/werefkin/drivers/macbook12-bluetooth-driver/build/bluetooth/hci_uart.ko] undefined!
ERROR: modpost: "aml_init" [/home/werefkin/drivers/macbook12-bluetooth-driver/build/bluetooth/hci_uart.ko] undefined!
make[3]: *** [scripts/Makefile.modpost:145: /home/werefkin/drivers/macbook12-bluetooth-driver/build/bluetooth/Module.symvers] Error 1
make[2]: *** [/usr/lib/modules/6.12.4-arch1-1/build/Makefile:1888: modpost] Error 2
make[1]: *** [Makefile:224: __sub-make] Error 2
make[1]: Leaving directory '/usr/lib/modules/6.12.4-arch1-1/build'
make: *** [Makefile:10: all] Error 2

```

any idea of how to resolve the issue?

--- a/patch_bluetooth/Makefile
+++ b/patch_bluetooth/Makefile
@@ -12,4 +12,5 @@ hci_uart-$(CONFIG_BT_HCIUART_BCM)	 += hci_bcm.o
 hci_uart-$(CONFIG_BT_HCIUART_QCA)	 += hci_qca.o
 hci_uart-$(CONFIG_BT_HCIUART_AG6XX)	 += hci_ag6xx.o
 hci_uart-$(CONFIG_BT_HCIUART_MRVL)	 += hci_mrvl.o
+hci_uart-$(CONFIG_BT_HCIUART_AML)	 += hci_aml.o
 hci_uart-objs                        := $(hci_uart-y)

